doc/src/manual/parallel-computing.md

```diff
@@ -299,11 +309,10 @@ julia> let B = B
 
 julia> @spawnat 2 whos();
 
-julia> 	From worker 2:	                             A    800 bytes  10×10 Array{Float64,2}
-	From worker 2:	                          Base               Module
-	From worker 2:	                          Core               Module
-	From worker 2:	                          Main               Module
-
+julia>  From worker 2:                               A    800 bytes  10×10 Array{Float64,2}
```

Comment:
the prompt prints before the results start coming in from the other worker, and the repl doesn't reset its position